metrics/README.md

```diff
@@ -0,0 +1,163 @@
+# Metrics
+
+This is a [Dropwizard Metrics](https://github.com/dropwizard/metrics) implementation 
```

Comment:
...yet the `README.md` is in `metrics`❓ I wonder whether this should be part of `./docs` or even the Scaladoc for the package `usi-metrics`:

```scala
package com.mesosphere.usi

/**
 * ...
 * == Metrics Types ==
 */
package metrics {}
```

Reply:
Hm, I'm not sure. I understand the idea but I find browsing READMEs in subdirectories a strange practice. A single point of entry for docs is much nicer. Also for us who will have to maintain all the docs and READMEs. Having a `./README.md`, `metrics/README.md`, `./dcos/metrics.md` and Scaladoc for `com.mesosphere.usi.metrics` seems too much. Where should we document what?

Reply:
As discussed offline - let's move forwards with READMEs for now and have a bigger discussion on docs structure, templates, content etc.